app/Policies/TagPolicy.php

```
@@ -0,0 +1,37 @@
+
+use App\Models\User;
+
+class TagPolicy extends BasePolicy
+{
+	public const CAN_LIST = 'canList';
+	public const CAN_EDIT = 'canEdit';
+
+	/**
+	 * Determine whether the user can list tags.
+	 */
+	public function canList(User $user): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (Unused Code Rules)

(UnusedFormalParameter)